### DISCUSSION TITLE
How and when copilot X can be used?

### DISCUSSION THREAD

**Discussion by wanglu2014** [community/community#53487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

 I am a professor with github copilot (education version). How and when I can use copilot X?
Thank you for your time.

**Comment by NJacksonDev** [comment]
Here is a good/short youtube video which briefly describes some of the main features of Copilot X.

https://www.youtube.com/watch?v=q1HZj40ZQrM
